.binstar.yml

```diff
@@ -0,0 +1,30 @@
+after_failure:
+- echo "After failure message!"
+after_script:
+- echo "ProtoCI build was a:$BINSTAR_BUILD_RESULT" | tee artifact1.txt
+after_success:
+- echo "ProtoCI BUILD SUCCESS"
+before_script:
+- echo "Prepare to install networkx PyYAML requests jinja2...."
+- conda install networkx PyYAML requests jinja2
+- echo "Prepare to clone conda and conda-build for metadata reading..."
+- git clone https://github.com/conda/conda-build
+- git clone https://github.com/conda/conda
+- export PYTHONPATH=$PYTHONPATH:./conda-build:./conda
+- echo "PYTHONPATH below should include ./conda and ./conda-build"
+- echo $PYTHONPATH
+
+build_targets:
+- conda
+engine:
+- python
+script:
+ - python build2.py ./ -buildall
+install_channels:
+- defaults
+- python
+package: ProtoCI
+platform:
+- osx-64
+- linux-64
+user: psteinberg
```

Comment:
I'm sort of +1 on Ryan's request.  On the one hand, we'll most likely simply create a build-bot account to dump auto-built files to, and never change this setting.  On the other, it would be nice to have it more dynamically settable.  It's OK with me if it is simply not easily doable to keep it hard-coded.